src/content-tags-drawer/ContentTagsDrawer.jsx

```diff
@@ -64,7 +65,7 @@ const ContentTagsDrawer = ({ id, onClose }) => {
     const handleEsc = (event) => {
       /* Close drawer when ESC-key is pressed and selectable dropdown box not open */
       const selectableBoxOpen = document.querySelector('[data-selectable-box="taxonomy-tags"]');
-      if (event.key === 'Escape' && !selectableBoxOpen) {
+      if (event.key === 'Escape' && !selectableBoxOpen && !blockingSheet) {
```

Comment:
I think it's simpler to do this instead: 

`const blockingSheet = (Object.keys(globalStagedContentTags).length + Object.keys(globalStagedRemovedContentTags).length) > 0;`

There doesn't seem to be any need to add another context, and state variable for something that can already be derived from existing info.

Reply:
I think it is better to leave it as context since it is used in two different places:
- To avoid close on click outside: https://github.com/openedx/frontend-app-course-authoring/pull/965/files#diff-44e5b0a241ffd4d6230ede25608e5392858869b1b6761b64f1daac7c18605ff2R21
- To avoid close on press Escape: https://github.com/openedx/frontend-app-course-authoring/pull/965/files#diff-8f2b1f4e041cee96dfc16c4c3da7631943407c989c2ea619594681f0c40e775bR68

I prefer it this way, because in the first case it is not possible to calculate it there, and it is better to calculate that value in [one place](https://github.com/openedx/frontend-app-course-authoring/pull/965/files#diff-dbee10a7547f7e8ce19a9b8879c445a48ed655dd1576eae1d794998423f3f936R355). 

> I think it's simpler to do this instead:
const blockingSheet = (Object.keys(globalStagedContentTags).length + Object.keys(globalStagedRemovedContentTags).length) > 0;

Furthermore, it could not be calculated in this way, because there is a case in which there are keys in the map, but they contain empty lists (add a tag, and remove the same tag later).